### DISCUSSION TITLE
AdGuard doesn't resolve adresses for me

### DISCUSSION THREAD

**Discussion by rabbitambulance** [AdguardTeam/AdGuardHome#6600] [Q&A]
Hi all,
I hope I’m in the right place with this problem, and I’d really appreciate any help.
I’ve installed AdGuard as an Add-on to my Home Assistant, started it and added the integration to my dashboard. Then I put my Home Assistant’s IP into my router as the DNS server, and now I can’t resolve any URLs anymore, from any of the devices on my network.
If I click on the setup guide tab of the adguard settings it says

```
AdGuard Home DNS server is listening on the following addresses:
127.0.0.1
```

I’ve tried adding my router as a persistent client in the client settings tab, but as far as i can tell that shouldn’t affect basic functionality? As you maybe can tell, I’m a little lost.

**Comment by ainar-g** [comment]
If I recall correctly, the HomeAssistant package adds its own configuration, so you're better off asking on their forums.  But generally, unless the package also sets up a proxy, an AGH instance on a router should listen to other additional addresses apart from `localhost`.
